pwiz_tools/Skyline/Model/Import.cs

```diff
@@ -3375,6 +3375,43 @@ public TransitionImportErrorInfo(string errorMessage, int? columnIndex, long? li
             Column = columnIndex + 1;   // 1 based column number for reporting to a user
             LineNum = lineNum;
         }
+
+        public bool Equals(TransitionImportErrorInfo other)

```

Comment:
Are the equals functions used? We try to stay away from the operator overrides in general. I imagine ReSharper just added them automatically for you now. Hopefully they can be turned off in the future. I have always used if (Equals(a, b)) on Skyline.

Reply:
I added these purely for use in the test, and yes, ReSharper generated that code.

Reply:
I'll remove the == and != overrides.